src/software/ai/navigator/path_planner/hrvo/hrvo_simulator.cpp

```diff
@@ -174,6 +287,30 @@ Vector HRVOSimulator::getRobotVelocity(unsigned int robot_id) const
     return Vector();
 }
 
+
+AngularVelocity HRVOSimulator::getRobotAngularVelocity(unsigned int robot_id) const
+{
+    auto hrvo_agent = robots.find(robot_id);
+    if (hrvo_agent != robots.end())
+    {
+        return hrvo_agent->second->getAngularVelocity();
+    }
+    LOG(WARNING) << "Angular velocity for robot " << robot_id
+                 << " can not be found since it does not exist in HRVO Simulator"
+                 << std::endl;
+    return AngularVelocity();
+}
+
+std::optional<Angle> HRVOSimulator::getRobotOrientation(RobotId robot_id) const
+{
+    auto hrvo_agent = robots.find(robot_id);
+    if (hrvo_agent != robots.end())
+    {
+        return hrvo_agent->second->getOrientation();
+    }
```

Comment:
lets have a log similar to `getRobotAngularVelocity`

Reply:
I was actually considering removing the other one in `getRobotAngularVelocity` as well, since these logs tend to spam a lot when the robot has not received a World, which results in HRVO sim not having the Agents.